From 7b736592c14d1e7ec0b2f035849d7cecf34d5f25 Mon Sep 17 00:00:00 2001
From: Mike Frysinger <vapier@gentoo.org>
Date: Wed, 11 Apr 2012 16:01:29 -0400
Subject: [PATCH] mmap64: include stdint.h for uint64_t

Some targets include this implicitly, but pull it in explicitly for those
which don't to fix building for them.

Signed-off-by: Mike Frysinger <vapier@gentoo.org>
---
 libc/sysdeps/linux/common/mmap64.c |    1 +
 1 files changed, 1 insertions(+), 0 deletions(-)

diff --git a/libc/sysdeps/linux/common/mmap64.c b/libc/sysdeps/linux/common/mmap64.c
index 3c97c84..cebf961 100644
--- a/libc/sysdeps/linux/common/mmap64.c
+++ b/libc/sysdeps/linux/common/mmap64.c
@@ -10,6 +10,7 @@
 #ifdef __UCLIBC_HAS_LFS__
 
 #include <errno.h>
+#include <stdint.h>
 #include <unistd.h>
 #include <sys/mman.h>
 #include <sys/syscall.h>
-- 
1.7.8.5

